Analysis/src/TypedAllocator.cpp

```diff
@@ -26,6 +26,8 @@ const size_t kPageSize = sysconf(_SC_PAGESIZE);
 
 #include <stdlib.h>
 
+#include <cstdint>
```

Comment:
```suggestion
#include <stdint.h>
#include <stdlib.h>
```

We do not use C++ namespaced types (like `std::uint8_t`), so it doesn't make sense to include C++ header here.
While C++ headers generally provide C definitions as well, it's all a mess in C++ standard, so best to use a C header.

Reply:
sorry for the mistake and thanks for having a look at it.